gwvolman/lib/dataone/metadata.py

```diff
@@ -214,6 +214,10 @@ def set_user_name(self, root, first_name, last_name):
         individual_name = ET.SubElement(root, 'individualName')
         ET.SubElement(individual_name, 'givenName').text = first_name
         ET.SubElement(individual_name, 'surName').text = last_name
+        if contact_info is not None:
+            user_directory = ET.SubElement(root, 'userId')
```

Comment:
`user_directory` is strange here, since the element is really `userId` which has a `directory` attribute.  Maybe `userid_elem` or something similar?